src/jaykumar/Assignment23.java

```diff
@@ -0,0 +1,40 @@
+/*Assignment - 23 : 14th Sep'2023
+Write a program to find count of Uppercase, lowercase, digits and special characters.
+
+String str = "TecHNo_cR6ediTs";
+output : Uppercase -> 5
+         Lowercase -> 8
+		 Digit -> 1
+		 Special char count -> 1*/
+		 
+package jaykumar;
+
+public class Assignment23 {
+	
+	void charCase(String input) {
+		int uCount=0;
+		int lCount=0;
+		int dCount=0;
+		int sCount=0;
+		for(int index=0;index<input.length();index++) {
+			if(Character.isUpperCase(input.charAt(index))) {
```

Comment:
you can write char ch = input.charAt(index) outside of conditional statement, so that you no need to rewrite again and again. you just need to write like Character.isUpperCase(ch)

Reply:
Sure, I'll use this in next assignments.